capstone/src/main/resources/static/user.js

```diff
@@ -393,26 +357,32 @@ function createCalendar() {
   });
 }
 
-/* helper function to create calendar event elements */
-function createCalendarEventElement(event, eventTime) {
-  const eventDisplay = document.createElement("div");
-  eventDisplay.setAttribute("class", "event general-container col");
-  const pElementTitle = document.createElement('p');
-  pElementTitle.innerText = event.eventTitle;
-  eventDisplay.appendChild(pElementTitle);
-  const pElementTime = document.createElement('p');
-  pElementTime.innerText = eventTime.toDateString();
-  eventDisplay.appendChild(pElementTime);
-  return eventDisplay;
+/* helper function to create calendar events */
+function createCalendarEvent(event, today, endDate, borderColor) {
+  var eventDate = new Date(event.eventDateTime);
+  if (eventDate.getTime() > today.getTime() && eventDate.getTime() < endDate.getTime()) {
+    var diff = eventDate.getDate() - today.getDate();
```

Comment:
I wonder how this is going to work if `eventDate` is the 1st of August, and today is the 30th of July.

Reply:
You are right I think this will cause an error.